### PR TITLE
Refactor release packaging to be DRYer

### PR DESCRIPTION
**Pull request by stevekuznetsov**
@smarterclayton WDYT?

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

### CODE REVIEW

File: hack/common.sh [hunk 1]
```diff
@@ -34,6 +34,8 @@ function os::build::host_platform_friendly() {
     echo "linux-32bit"
   elif [[ $platform == "linux/amd64" ]]; then
     echo "linux-64bit"
+  elif [[ $platform == "linux/ppc64le" ]]; then
+    echo "linux-ppc64le"
```
Comment (smarterclayton) [inline hack/common.sh]: linux-powerpc64

**Comment by smarterclayton** [comment]
Otherwise fine

**Comment by stevekuznetsov** [comment]
@smarterclayton updated

**Comment by stevekuznetsov** [comment]
[merge]

**Comment by smarterclayton** [comment]
Missed a spot

On Feb 2, 2017, at 9:37 AM, OpenShift Bot <notifications@github.com> wrote:

continuous-integration/openshift-jenkins/merge Running (
https://ci.openshift.redhat.com/jenkins/job/test_pr_origin/13506/) (Base
Commit: cfb50d9
<https://github.com/openshift/origin/commit/cfb50d9156c6f5991a868a8d272e6b5503bebbae>
)

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/12723#issuecomment-276974346>,
or mute
the thread

**Comment by stevekuznetsov** [comment]
@smarterclayton not sure what that response means -- abort merge?

**Comment by smarterclayton** [comment]
The integration tests are failing because of a change you made

**Comment by tdawson** [comment]
[merge]

**Pull request merged**